rasa/telemetry.py

```diff
@@ -482,7 +481,6 @@ def _default_context_fields() -> Dict[Text, Any]:
             "directory": _hash_directory_path(os.getcwd()),
             "python": sys.version.split(" ")[0],
             "rasa_open_source": rasa.__version__,
-            "gpu": len(tf.config.list_physical_devices("GPU")),
```

Comment:
this also requires updating the example in `telemetry.mdx`

Reply:
:+1: